documentation/reference/releases/index.md

```diff
@@ -4,30 +4,31 @@
 
 | Release Date | Release                                     |
 | ------------ | ------------------------------------------- |
-| 2024 Mar 5   | [1.2.18 Minor Release](2024-mar-1.2.18.md)  |
+| 2024 Mar 5   | [1.2.18 Minor Release](/reference/releases/2024-mar-1.2.18.md)  |
+| 2024 Mar 5   | [1.2.18 Minor Release](/reference/releases/2024-mar-1.2.18.md)  |
```

Comment:
Added twice - remove one.

Reply:
Thanks. Must have been an artifact from pulling in updates on test.